src/index.js

```diff
@@ -9,28 +9,8 @@ import { renderToStaticMarkup } from './react-compat';
  * @class Enzyme
  */
 
-let jsdom;
-try {
-  require('jsdom'); // could throw
-  jsdom = require('mocha-jsdom');
-} catch (e) {
-  // jsdom is not supported...
-}
-
 export let sinon = Sinon.sandbox.create();
 
-export function describeWithDOM(a, b) {
```

Comment:
the only problem is that this makes it a breaking change - it'd be nice if you re-exported the now-extracted `describeWithDOM`, making this PR a minor change only